src/apphosting/index.ts

```diff
@@ -361,3 +361,19 @@ export async function orchestrateRollout(
   }
   return { rollout, build };
 }
+
+/**
+ * Deletes the given backend. Polls till completion.
+ */
+export async function deleteBackend(
```

Comment:
nit: `deleteBackendAndPoll()` if you want to stay consistent with ``createBackendAndPoll()` above